### ISSUE TITLE
dellemc_powerscale_filesystem does not honor the recursive paramater during deletes

### ISSUE THREAD

**Issue by rbenigno** [dell/ansible-powerscale#6]
The dellemc_powerscale_filesystem module is not able to delete file systems unless they are empty.

Would like to do something like the following with the 'recursive' parameter from the playbook being honored for deletes:

```
    - name: Delete target file systems
      dellemc_powerscale_filesystem:
       onefs_host: "{{synciq_target}}"
       verify_ssl: "{{verify_ssl}}"
       api_user: "{{api_user}}"
       api_password: "{{api_password}}"
       access_zone: "{{access_zone}}"
       path: "{{item.path}}"
       state: "absent"
       recursive: True
      loop: "{{smb_shares}}"
```

The module has a 'recursive' parameter, and it looks like the SDK also includes a recursive option on 'delete_directory'.

Thank you!

**Comment by anupamaloke** [comment]
@rbenigno, I guess this restriction was put in place as per the design.  There could be a case where an admin user accidently choose to recursive delete and end up losing a large amount of user data. Someone can simply set ```state: absent``` on the ```/ifs``` folder and that way lose everything that they have on the cluster – imagine that! So the directory is not deleted if it is exported or not empty.

**Comment by anupamaloke** [comment]
closing the issue as it has been addressed in v1.5.0.
